TYPO3.Media/Classes/TYPO3/Media/ViewHelpers/ImageViewHelper.php

```diff
@@ -104,7 +104,7 @@ public function initializeArguments()
     /**
      * Renders an HTML img tag with a thumbnail image, created from a given image.
      *
-     * @param ImageInterface $image The image to be rendered as an image
+     * @param ImageInterface|null $image The image to be rendered as an image
```

Comment:
In my test flow did not understand  the ``|null`` properly. If i change this line the change works as expected.